package.json

```diff
@@ -2,9 +2,10 @@
   "name": "bitbox02-api",
   "version": "0.8.1",
   "description": "BitBox02 JavaScript library – To enable communication from the browser to the BitBox02 hardware wallet",
-  "main": "src/index.js",
+  "main": "lib/index.js",
```

Comment:
We should consider to add module fiel that points to ES6, as our current module only ships ES6 and we should keep supporting it.

Reply:
why keep supporting it? the usage of the API stays the same. does it matter much if it is ES5 or ES6 that is used by clients?

I checked `module`, seems like a huge mess: https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for

Reply:
that SO is from 4 years, 5 months ago

Reply:
Realized we actually never fully used ES6, internally it used ES5 anyways, see https://github.com/digitalbitbox/bitbox02-api-js/pull/52#issuecomment-904420806